CHANGES

```diff
@@ -4,6 +4,10 @@ Release 5.2.0 (in development)
 Dependencies
 ------------
 
+* #10356: Sphinx now uses declarative metadata with ``pyproject.toml`` to
+  create packages, using PyPA's ``build`` project as a build backend. Patch by
```

Comment:
Just noticed this in the ChangeLog — `build` is actually a frontend, `flit_core` is the backend.

Reply:
Fixed, thank you.

A

Reply:
Thanks!